addons/website_sale/tests/test_website_sale_visitor.py

```diff
@@ -52,8 +52,9 @@ def test_create_visitor_on_tracked_product(self):
         self.assertEqual(len(new_visitors), 1, "No visitor should be created after visiting another tracked product")
         self.assertEqual(len(new_tracks), 2, "A track should be created after visiting another tracked product")
 
-    def test_recently_viewed_company_changed(self):
-        # Test that, by changing the company of a tracked product, the recently viewed product do not crash
+    def test_dynamic_filter_newest_products(self):
```

Comment:
Don't change a method name in stable, even in tests.
Most likely not going to cause any problem, but better safe than sorry, someone could have a custo in the codebase for which he needs to patch this test for some reason.
Let's not try to guess how likely it is and let's just stick to the usual rule, don't change signature or method name in stable, test or not 
It is not the end of the world in tests, sure, but that's done for no reason.

Reply:
Used to not do it outside the tests, but it totally makes sense to have same logic in tests, thanks!